tools/deviceGen/input/device_features.json

```diff
@@ -0,0 +1,2666 @@
+{
+    "vulkan10core": [
+        {
+            "groupComment": [
+                "Vulkan 1.0 Core"
+            ]
+        },
+        {
+            "type": "bool",
+            "name": "robustBufferAccess",
+            "value": false,
+            "comment": "widely supported but has performance overhead, so remains an optional feature to enable"
+        },
+        {
+            "groupComment": [
+                "[REQUIRE] Roadmap 2022 requires support for these, device support is ubiquitous and enablement is unlikely to harm performance"
+            ]
+        },
+        {
+            "type": "bool",
+            "name": "fullDrawIndexUint32",
+            "value": true,
+            "expose": false
+        },
+        {
+            "type": "bool",
+            "name": "imageCubeArray",
+            "value": true,
+            "expose": false
+        },
+        {
+            "type": "bool",
+            "name": "independentBlend",
+            "value": true,
+            "expose": false
+        },
+        {
+            "groupComment": [
+                "I have no clue if these cause overheads from simply being enabled"
+            ]
+        },
+        {
+            "type": "bool",
+            "name": "geometryShader",
+            "value": false
+        },
+        {
+            "type": "bool",
+            "name": "tessellationShader",
+            "value": false
+        },
```

Comment:
IMHO, structurally these should be in the same brace as their `groupComment`

Reply:
Done